src/urls.js

```diff
@@ -51,6 +51,18 @@ class Urls {
     return `${this.groupsBase()}/${id}`;
   }
 
+  getGroupActivity(id, query) {
+    return `${this.group(id)}/activity${this.queryString(query)}`;
+  }
+
+  getGroupAdmins(id) {
+    return `${this.groupsBase()}/${id}/admins`;
```

Comment:
`${this.groupsBase()}/${id}` in `getGroupAdmins` and `getGroupMembers` can also be replaced by `${this.group(id)}`

Reply:
Ah, good call!